src/sentry/incidents/grouptype.py

```diff
@@ -26,22 +27,42 @@
 
 @dataclass
 class MetricIssueEvidenceData(EvidenceData):
-    alert_id: int
+    alert_id: int  # is this supposed to be detector id?
 
 
 class MetricAlertDetectorHandler(StatefulGroupingDetectorHandler[QuerySubscriptionUpdate, int]):
     def create_occurrence(
         self, group_key: DetectorGroupKey, new_status: PriorityLevel
     ) -> tuple[DetectorOccurrence, dict[str, Any]]:
-        # Returning a placeholder for now, this may require us passing more info
-        occurrence = DetectorOccurrence(
-            issue_title="Some Issue Title",
-            subtitle="An Issue Subtitle",
-            type=MetricIssue,
-            level="error",
-            culprit="Some culprit",
-        )
-        return occurrence, {}
+
+        try:
+            assignee = parse_and_validate_actor(
+                self.detector.created_by_id, self.detector.project.organization_id
+            )
+        except Exception:
+            assignee = None
+
+        # XXX: can we pass data_packet through here? we need the aggregate value and the source ids to generate the title
+        # title = construct_title(incident.alert_rule, incident.status)
+        # TODO rewrite construct_title to not take incident data
+        title_level = "Critical" if new_status == PriorityLevel.HIGH else "Warning"
+        title = f"{title_level}: 100 events in the last 5 mins"
+
+        return (
+            DetectorOccurrence(
+                issue_title=self.detector.name,
+                subtitle=title,
+                resource_id=None,
+                evidence_data={},  # what's supposed to be in here?
```

Comment:
the majority of this will be set by default in the detector itself -- this is any additional data you want to see in the UI. (these are the values that will be set by default: https://www.notion.so/sentry/Evidence-Data-1c78b10e4b5d8076b103e552a53fd196)